app/models/piece.rb

```diff
@@ -16,8 +16,86 @@ def self.types
     %w(Pawn Rook Knight Bishop Queen King)
   end
 
-  def valid_move? x_coordinate, y_coordinate
+  def valid_move?(x_coordinate, y_coordinate)
```

Comment:
I only changed this line because Rubocop was complaining about it. After this I tried to leave code I didn't write alone unless I had to change it.